src/librustdoc/html/render/mod.rs

```diff
@@ -1165,7 +1165,7 @@ fn render_deref_methods(
     }
 }
 
-fn should_render_item(item: &clean::Item, deref_mut_: bool, cache: &Cache) -> bool {
+fn should_render_item(item: &clean::Item, deref_mut_: bool, cx: &Context<'_>) -> bool {
```

Comment:
Same nit as on the other PR, this could take a tcx instead.

Reply:
Ah, thanks for catching that. I forgot to cherry-pick the commit from the other PR that fixed it. Done.